pyquil/operator_estimation.py

```diff
@@ -814,8 +814,67 @@ def serializable(self):
         }
 
 
+def _generate_experiment_programs(tomo_experiment: TomographyExperiment,
+                                  active_reset: bool = False
+                                  ) -> Tuple[List[Program], List[List[int]]]:
```

Comment:
sorry for flipflopping a bit -- lets go with this style when function signatures cannot fit on one line (using [Black](https://github.com/psf/black/blob/master/README.md) as a reference)

```python
def very_important_function(
    template: str,
    *variables,
    file: os.PathLike,
    engine: str,
    header: bool = True,
    debug: bool = False,
) -> None:
```